build/configs/rtl8721csm/loadable_apps/defconfig

```diff
@@ -199,8 +199,11 @@ CONFIG_BOOT_RUNFROMFLASH=y
 #
 # Boot Memory Configuration
 #
-CONFIG_RAM_KREGIONx_START="0x02000000,"
-CONFIG_RAM_KREGIONx_SIZE="3145728,"
+CONFIG_RAM_KREGIONx_START="0x10005000,0x02000000,0x02100000"
+CONFIG_RAM_KREGIONx_SIZE="466944,1048576,2097152,"
```

Comment:
Could you pls increase the kernel heap size in PSRAM to cover the entire available area?